vscode/src/chat/chat-view/SimpleChatPanelProvider.ts

```diff
@@ -561,11 +563,7 @@ export class SimpleChatPanelProvider implements vscode.Disposable, ChatSession {
 
     private handleAbort(): void {
         this.cancelInProgressCompletion()
-        telemetryService.log(
-            'CodyVSCodeExtension:abortButton:clicked',
-            { source: 'sidebar' },
-            { hasV2Event: true }
-        )
+        telemetryService.log('CodyVSCodeExtension:abortButton:clicked', { hasV2Event: true })
```

Comment:
```suggestion
        telemetryService.log('CodyVSCodeExtension:abortButton:clicked', {}, { hasV2Event: true })
```

or `hasV2Event` gets logged as a property